### DISCUSSION TITLE
Disable TOC on the homepage

### DISCUSSION THREAD

**Discussion by camilledervout** [dotnet/docfx#9116] [Q&A] [answered]
Is it possible to disable the TOC on the homepage while keeping the navigation bar?
https://dotnet.github.io/docfx/docs/table-of-contents.html#navigation-bar

**Comment by filzrev** [comment, marked as answer]
If you are using `default` template.
Set [`_disableToc` metadata](https://dotnet.github.io/docfx/docs/template.html?tabs=default) to homepage. 

If you are using `modern` template.  
Try to set [`_layout: landing` metadata](https://dotnet.github.io/docfx/docs/template.html?tabs=modern) to homepage.

This settings hide both `toc` and `affix` area.
If you need to hide `toc` area only. You needs to create custom template.

**Reply by KalleOlaviNiemitalo** [reply]
The documentation of the metadata is a bit confusing.  There is a list in [Predefined Metadata](https://dotnet.github.io/docfx/reference/docfx-json-reference.html#predefined-metadata) but then separate lists for each template.  And [YAML header](https://dotnet.github.io/docfx/docs/markdown.html#yaml-header) links to [predefined metadata](https://dotnet.github.io/docfx/docs/config.html#predefined-metadata) which does not exist at that URL any more.

**Comment by camilledervout** [comment]
Thank you, it works perfectly. I added this code in docfx.json file

```json
"fileMetadata": {
      "_layout": {
        "index.md": "landing"
      }
    }
```
